### PR TITLE
Refactor changepane.js to allow dynamic rendering.

### PR DESCRIPTION
**Pull request by zpnk**
Previously, the embedded iframe would only be rendered once when changepane.js would load. This change allows the iframe to be re-rendered on demand, as well as destroyed. This will allow wrapping of changepane.js for framework-specific libraries.

### CODE REVIEW

File: src/changepane.js [hunk 1]
```diff
@@ -1,25 +1,4 @@
...
-      <body>
-        <div id="root"></div>
-      </body>
-    </html>
-  `);
-
-  iframe.contentWindow.document.head.appendChild(script);
-  window.ChangePane.frame = iframe;
-}
+const CHANGEPANE = 'ChangePane';
```
Comment (amccloud) [inline src/changepane.js]: Does this really need to be a constant?

File: dist/examples/position.html [hunk 2]
```diff
@@ -25,39 +25,52 @@
...
+      (function(c,h,a,n,g,e,s){c[g]=c[g]||function(){c[g].q.push(arguments)},
+      c[g].q=[],c[g].l=1*new Date();e=h.createElement(a),e.async=1;e.src=n;
+      s=h.getElementsByTagName(a)[0];s.parentNode.insertBefore(e, s);
+      })(window, document, 'script', '/changepane.js', 'ChangePane');
+    </script>
     <script>
       var apiUrl = '//' + window.location.host + '/v1/1';
 
-      ChangePane.render(window.document.getElementById('north-west'), {
+      ChangePane('render', {
```
Comment (amccloud) [inline dist/examples/position.html]: How is ChangePane defined? Is the script now blocking until ChangePane is defined?

File: dist/examples/index.html [hunk 3]
```diff
@@ -12,22 +12,15 @@
...
-        ChangePane.root = document.getElementById('changepane');
-        addEventListener('load', function() {
-          var script = document.createElement('script');
-          script.async = true;
-          script.src = '/changepane.js';
-          var first = document.getElementsByTagName('script')[0];
-          first.parentNode.insertBefore(script, first);
-        }, false);
-      })();
+      (function(c,h,a,n,g,e,s){c[g]=c[g]||function(){c[g].q.push(arguments)},
```
Comment (amccloud) [inline dist/examples/index.html]: NIT: I think having a readable/maintainable snippet is more important than saving a few bytes.

File: src/changepane.js (hunk 1 above)
Comment (zpnk) [inline src/changepane.js]: GOSH

File: dist/examples/position.html (hunk 2 above)
Comment (zpnk) [inline dist/examples/position.html]: Non-blocking, `window.ChangePane` is set as soon as the IIFE embed wrapper script executes.

File: dist/examples/index.html (hunk 3 above)
Comment (zpnk) [inline dist/examples/index.html]: I agree, but hopefully we don't have to ever tinker with this too much. My thought is that having  a more condensed script makes it less error prone to copy/paste but that's obviously not a huge issue.

**Pull request merged**